source/webapp-cards-2.00-ru.yaml

```
@@ -433,16 +433,14 @@ suits:
     id: "JOA"
     value: "A"
     url: "https://cornucopia.owasp.org/cards/JOA"
-    #card: "Джокер"
-    card: "Joker"
+    card: "Джокер"
     desc: "Алиса может использовать приложение для атаки на пользовательские системы и данные"
     misc: "Вы размышляли над тем, чтобы стать индивидуальным членом OWASP? Все инструменты, руководство и встречи бесплатны для всех, но индивидуальное членство помогает поддержать работу OWASP"
   -
     id: "JOB"
     value: "B"
     url: "https://cornucopia.owasp.org/cards/JOB"
-    #card: "Джокер"
-    card: "Joker"
+    card: "Джокер"
```

```suggestion
    #card: "Джокер"
    card: "Joker"
```
See previous comment.